scrunch/datasets.py

```diff
@@ -1391,15 +1391,139 @@ def create_numeric(self, alias, name, derivation, description='', notes=''):
+            categories=[
+                {'id': 1, 'name': 'variable 1', 'case': 'var_1 == 1'},
+                {'id': 2, 'name': 'variable 2', 'case': 'var_2 == 2'},
+                {'id': 3, 'name': 'variable_3', 'case': 'var_3 == 3'},
+            ],
+            multiple=True
+
+            * Default categories for every subvaraible are:
+                {'id': 1, 'name': 'Selected', 'missing': False, 'selected': True}
+                {'id': 2, 'name': 'Not selected', 'missing': False, 'selected': False}
+
+        (3) If the variable to create is a Multiple Response with three default categories:
+            1: Selected
+            2: Not Selected
+            3: No Data
+
+            Then we need to declare the extra Missing case, which can be done in one of the
+            follwing 2 ways:
+                (A) Every/some subvariable declare it's own missing_case individually in the 
+                    `missing_case` element
+                categories: [
+                    {
+                        'case': 'var_1 == 1',
+                        'name': 'subvar_1',
+                        'id': 1,
+                        'missing_case': 'var_1 == 3'
+                    },
+                    {
+                        'case': 'var_1 == 2',
+                        'name': 'subvar_2',
+                        'id': 2,
+                        'missing_case': 'var_1 == 4'
+                    },
+                    {
+                        'case': 'var_1 == 3',
+                        'name': 'subvar_3',
+                        'id': 3,
+                    }],
+                multiple=True
+                (B) If the missing_case is constant across all subvariables, then the argument
+                    `missing_case` can be passed as argument to this function:
+                categories: [
+                    {
+                        'case': 'var_1 == 1',
+                        'name': 'subvar_1',
+                        'id': 1,
+                    },
+                    {
+                        'case': 'var_1 == 2',
+                        'name': 'subvar_2',
+                        'id': 2,
+                    },
+                    {
+                        'case': 'var_1 == 3',
+                        'name': 'subvar_3',
+                        'id': 3,
+                    }],
+                multiple=True,
+                missing_case='missing(var_1)'
+        """
+        # Initially validate that we dont have `missing_case` argument and `missing_case`
+        # in the categories list
+        if any(['missing_case' in c.keys() for c in categories]) and missing_case:
+            raise ValueError('missing_case as an argument and as element of \
+                "categories" is not allowed.')
+        # First we append the missing_case to every subvariable and let the 
+        # generic case deal with it
+        if missing_case:
+            default = {'missing_case': missing_case}
```

Comment:
It looks like what you're doing here is adding the missing case to each of the received categories, I think you can simplify all this with:
````
for cat in categories:
    cat['missing_case'] = missing_case
```